bower.json

```diff
@@ -9,7 +9,9 @@
     "angular-resource": "^1.4.0",
     "angular-route": "^1.4.0",
     "angular-sanitize": "^1.4.0",
-    "angular-touch": "^1.4.0"
+    "angular-touch": "^1.4.0",
+    "moment": "~2.13.0",
```

Comment:
You've added momentJS to the project deps, but have never used it. Remove it or utilize it.